pyproject.toml

```diff
@@ -6,16 +6,16 @@ authors = ["Ben Beecher <ben@lightmatter.com>"]
 
 [tool.poetry.dependencies]
 python = "^3.9"
-cookiecutter = "^1.7.3"
-isort = "^5.10.1"
-Sphinx = "^5.1.1"
-furo = "^2022.9.15"
-
-[tool.poetry.dev-dependencies]
+cookiecutter = "^2.1.1"
```

Comment:
I think we want to keep this at 1.7.x for now - there's some open issue with cruft

Reply:
I think I'd need at minimum v2.0.1 (not completely sure, hard to pin down exactly when this change was released) so I can use the `--accept-hooks` flag.

PR: https://github.com/cookiecutter/cookiecutter/pull/992